website/source/docs/concepts/policies.html.md

```diff
@@ -99,6 +111,31 @@ capabilities. These mappings are as follows:
 
   * `read` - `["read", "list"]`
 
+## Permissions
+
+Paths offer an optional `permissions` field for fine-grain control over the
+parameters and values a path is able to write. The capabilities associated with
+this path take precedence over permissions on parameters.
+
+  * `allowed_parameters` - Acts as a whitelist. Setting to "*" will allow a
```

Comment:
Sadly because it's markdown this confuses the italics checking. Should be `*` instead of "*". (Or maybe for clarity, `"*"`)